src/platform/intel/ace/platform.c

```diff
@@ -74,6 +78,74 @@ int platform_boot_complete(uint32_t boot_message)
 	return ipc_platform_send_msg(&msg);
 }
 
+/* address where zephyr PM will save memory during D3 transition */
+#ifdef CONFIG_ADSP_IMR_CONTEXT_SAVE
+extern void *global_imr_ram_storage;
+#endif
+
+/**
+ * @brief Notifier called before every power state transition.
+ * Works on Primary Core only.
```

Comment:
This code is not doing anything for secondary cores but it will be executed on them. This comment need to be changed or removed.